reproducibility_project/src/utils/trajectory_conversion.py

```diff
@@ -0,0 +1,74 @@
+"""Convert Cassandra trajectories to gsd."""
+import io
+from pathlib import Path
+
+import gsd.hoomd
+import mbuild as mb
+import numpy as np
+import parmed
+
+
+def cassandra2gsd(Hpath, xyzpath, gsdpath, species_list):
```

Comment:
A little bit of info in the docstring about these parameters would be helpful, particularly species_list--I assume it is a list of pmd.Structures.

Reply:
I agree, I just haven't added that part yet.

Reply:
I've added it now.